frontend/app/styles/app.scss

```diff
@@ -0,0 +1,315 @@
+$break-middle: 768px;
+
+$light-grey: #E3E3E3;
+
+$dark-grey: #252525;
+
+
+//Sizes
+
+.w-20{
+  width: 20%;
+}
+
+
+.bg-grey{
+  background-color: $light-grey;
+}
+
+.border-1{
+  border: 1px solid $light-grey;
+}
+.border-2{
+  border: 2px solid $light-grey;
+}
+
+.border-d-2{
+  border-bottom: 2px solid $light-grey;
+}
+a{
+  color: inherit;
+  text-decoration: none;
+  background-color: transparent;
+  &:hover{
+    color: #0056b3;
+    text-decoration: none !important;
+  }
+}
+
+.c-grey{
+  color: grey;
+}
+.bg-white{
+  background-color: white;
+}
+
+//-----------------------Top-bar and Navbar------------------------------------
+#top-bar{
+  background-color: $dark-grey;
+}
+#nav-items{
+  flex-grow: 0;
+}
+
+#page-nav{
+
+  @media screen and(min-width:576px) {
```

Comment:
Ako koristiš bootstrap i moraš pisati ovakve stvari, treba odmah da ti je jasno da nešto ne radiš kako treba.